tests/README.md

```diff
@@ -16,7 +16,27 @@ CONNECTION_PARAMETERS = {
     'warehouse': '<warehouse>',
 }
 ```
-Note that this file will be ignored by git, so you do not need to worry about check in your secret.
+Snowpark pandas API also offers the convenience of implicit session creation from a configuration file.
+This eliminates the need to explicitly create a Snowpark session in your code, allowing you to write your pandas code just as you would normally.
```

Comment:
Rather than duplicating default connection instructions, I suggest that we link to here: https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-connect#setting-a-default-connection

Reply:
Good idea, done.